packages/node_modules/brightspace-auth-keys-redis-store/package.json

```diff
@@ -14,7 +14,7 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "istanbul cover --root src mocha -- --recursive ./test"
+    "test": "istanbul cover --root src ../../../node_modules/mocha/bin/mocha -- --recursive ./test"
```

Comment:
Did it really need this?

Reply:
To work in Windows it seems to be necessary. https://github.com/gotwarlost/istanbul/issues/90

Reply:
Alright cool. nyc may not have the same problem, so will look at fixing it afterward